solutions/usaco-992.mdx

```diff
@@ -9,8 +9,12 @@ author: Óscar Garries, Neo Wang
 
 [Official Analysis](http://www.usaco.org/current/data/sol_wormsort_silver_jan20.html)
 
+**Time Complexity:** $\mathcal{O}(N\log M)$ <!-- Not quite sure if the variables are right.-->
```

Comment:
should be $\mathcal O((N+M)\log \max w_i)$

Reply:
Thanks!